File: wowsca/src/main/java/jwt/validation/wowsca/model/Partes.java
package jwt.validation.wowsca.model;

import java.util.ArrayList;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Partes {
    private ArrayList<UsuarioGrupoDespesa> partes;
    private int idDespesa;
}

File: Front/src/app/model/partes.ts
import { UsuarioGrupoDespesa } from "./usuarioGrupoDespesa";

export class Partes{
  partes?: UsuarioGrupoDespesa[];
  idDespesa?: number;
  constructor(partes?: UsuarioGrupoDespesa[], idDesp?: number){
    this.partes = partes;
    this.idDespesa = idDesp;
  }
}
